go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	// github.com/strangelove-ventures/packet-forward-middleware/v2 v2.1.1
 	github.com/stretchr/testify v1.7.1
 	github.com/tendermint/tendermint v0.35.2
-	github.com/tendermint/tm-db v0.6.7
+	github.com/tendermint/tm-db v0.6.8-0.20220519162814-e24b96538a12
```

Comment:
it is intentional, because then we can make gaia support badgerdb v3.

Reply:
Hi, lets put the unreleased tm-db version in replace for now ? since main anyway has unreleased version of ibc in replace for rho-working-in-progress.